scripts/gceslave.sh

```diff
@@ -1,18 +1,18 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
+set -euxo pipefail
+
+export CLOUDSDK_CORE_PROJECT=${CLOUDSDK_CORE_PROJECT-${GOOGLE_PROJECT}}
```

Comment:
Done.

On Mon, Aug 29, 2016 at 1:25 PM, Peter Mattis <notifications@github.com>
wrote:

> In scripts/gceslave.sh
> <https://github.com/cockroachdb/cockroach/pull/8892#discussion_r76648077>:
>
> > @@ -1,18 +1,18 @@
> >  #!/usr/bin/env bash
> >
> > -set -euo pipefail
> > +set -euxo pipefail
> > +
> > +export CLOUDSDK_CORE_PROJECT=${CLOUDSDK_CORE_PROJECT-${GOOGLE_PROJECT}}
>
> Can we default the project to cockroach-$(whoami)?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/8892/files/5ecfd4b9a48d0e9fd1ee1adf9631572684263c0b#r76648077>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABdsPK_Yb2Zklo61RRlfGNk_TjQWFIeZks5qkxYBgaJpZM4JvsYK>
> .
>